Classes/Neos/Neos/Ui/Fusion/Helper/NodeInfoHelper.php

```diff
@@ -74,7 +74,11 @@ public function renderNode(NodeInterface $node, ControllerContext $controllerCon
             $nodeInfo['uri'] = $this->uri($node, $controllerContext);
         }
 
-        foreach ($node->getChildNodes() as $childNode) {
+        $documentChildNodes = $node->getChildNodes($this->baseNodeType);
+        $contentChildNodes = $node->getChildNodes('!Neos.Neos:Document');
```

Comment:
Right, so this *should* include any other child nodes even if they are not rendering anything (like for example the FormElement definition NodeTypes I'm working on)? If so it might be clearer to rename `$contentChildNodes` to `$remainingChildNodes`!?